test/classes/initializers/postInit/throwing-superclass3.chpl

```diff
@@ -0,0 +1,33 @@
+interface dummy { }
+interface dummy2 { }
+
+class Foo {
+  var x: int;
+
+  proc postinit() throws {
+    if (x > 5) {
+      throw new Error("x too large");
+    }
+  }
+}
+
+class Bar : dummy, Foo, dummy2 {
+  proc init(x: int) {
+    super.init(x=x);
+  }
+
+  proc postinit() {
+    writeln("In Bar's postinit");
+  }
+}
```

Comment:
I'd be interested to see throwing-supersuperclass tests where `Bar` doesn't have an explicit `postinit()`, but  `class Baz : Bar` does have one that throws / doesn't throw.

Reply:
For the test:

```chapel
class Foo {
  proc postinit() throws {
    throw new Error();
  }
}

class Bar: Foo {
}

class Baz: Bar {
  proc postinit() /* throws */ {
  }
}

var c = new Baz();
```

* If the `throws` remains commented out, we get an uncaught error at the postinit() line due to the compiler's implicit call to `super.init()`
* If it's uncommented, the uncaught error is at the call to `new Baz()`

I think these are both as expected

I'm not sure that adding the tests to this PR would add much more coverage than we already have in this PR since it's basically checking to see that the compiler properly injects throwing postinit() calls (covered by `throwing-superclass.chpl`), and that errors are caught either at the original call (covered by `throwing.chpl`) or at the non-throwing postinit() (covered by `throwing-superclass3.chpl`).  Let me know if you disagree, though, or see something this is covering that I've missed.